index.html

```diff
@@ -0,0 +1,165 @@
+          <li class="links"><a href="about.html">About </a></li>
+          <li class="links"><a href="#">Program</a></li>
+          <li class="links"><a href="#">Join</a></li>
+          <li class="links"><a href="#">Sponsor</a></li>
+          <li class="links last-link"><a href="#">News</a></li>
+          <li class="links focused"><a href="#">CC Campaign</a></li>
+        </ul>
+      </div>
+    </header>
+    <main>
+      <!--Intro Section-->
+      <section id="intro" class="intro">
+        <h2>"Hello! Sharing world"</h2>
+        <h1>creative commons global summit 2022</h1>
+        <p class="bio">
+          A joyful celebration believing in the value of openness and sharing,
+          creating a positive change with people from over 80 countries is
+          taking place in October, in Korea.
+        </p>
+        <p class="time">2022.10.15(thu) ~ 16(fri)</p>
+        <p class="address">
+          @ National Museum of Korea, Art Center Nabi and more
+        </p>
+      </section>
+
+      <!--Main Program Section-->
+      <section id="main-program">
+        <p class="title">Main Program</p>
+        <hr />
+        <div class="main-program">
+          <div class="lecture">
+            <img src="./assets/logos/p5.png" alt="lecture logo" />
+            <h3>Lecture</h3>
+            <p>
+              listen to the speakers from various countries about the messeges
+              of sharing and opening.
+            </p>
+          </div>
+          <div class="ccexhibition">
+            <img src="./assets/logos/p1.png" alt="cc Exhibition logo" />
+            <h3>CC Exhibition</h3>
+            <p>
+              Appreciate various creations applying CC license of artists,
+              organized from Art Center Nabi.
+            </p>
+          </div>
+          <div class="forum">
+            <img src="./assets/logos/p2.png" alt="Forum logo" />
+            <h3>Forum</h3>
+            <p>
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="workshop">
+            <img src="./assets/logos/p3.png" alt="workshop logo" />
+            <h3>Workshop</h3>
+            <p>
+              try creating your own work using open source license rather than
+              just watching at it.
+            </p>
+          </div>
+          <div class="ccignite">
+            <img src="./assets/logos/p4.png" alt="cc ignite logo" />
+            <h3>CC Ignite</h3>
+            <p>
+              Get opportunities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+        <p class="desk">See the whole program</p>
+        <p class="mob">Join CC Global Summit 2022</p>
+      </section>
+      <!--Featured Speakers Section-->
+      <section id="featured-speaker">
+        <div id="speakers"></div>
```

Comment:
Please ensure that the pages look almost identical to the original design.

Your Implementation | Figma Design
:---:|:---:
 
![your_design](https://user-images.githubusercontent.com/10332499/153335411-3f63bf2f-bebf-404b-92b6-a1fd1f31f69f.PNG)  | ![design](https://user-images.githubusercontent.com/10332499/153335427-210da551-691e-42a6-a634-32b71df78bde.PNG)

Reply:
yes, you right
I did solve  the issue